app/src/main/res/values-is/strings.xml

```diff
@@ -210,8 +210,8 @@
   <string name="location_permission_rationale_nearby">Heimild þarf til að birta lista yfir staði í nágrenninu</string>
   <string name="get_directions">FÁ LEIÐSÖGN</string>
   <string name="read_article">LESA GREIN</string>
-  <string name="notifications_welcome">Velkomin á Wikimedia Commons, %s! Við erum ánægð með að þú skulir vera hérna.</string>
-  <string name="notifications_talk_page_message">%s skildi eftir skilaboð á spjallsíðunni þinni</string>
+  <string name="notifications_welcome">Velkomin á Wikimedia Commons, %1$s! Við erum ánægð með að þú skulir vera hérna.</string>
+  <string name="notifications_talk_page_message">%1$s skildi eftir skilaboð á spjallsíðunni þinni</string>
   <string name="notifications_thank_you_edit">Takk fyrir að hafa gert breytingar</string>
-  <string name="notifications_mention">%s minntist á þig á %s.</string>
+  <string name="notifications_mention">%1$s minntist á þig á %2$s.</string>
```

Comment:
One thing I'd be worried about is that in some languages the order might be switched because of a word order different from English. I think this set of changes assumes that all languages have the same word order.

@Nikerabbit If we add' fuzzy="true"' here in the repository, will the bot at TranslateWiki.Net apply the status of those messages into the wiki? If yes, I think that's what we want to do.

Reply:
I agree with that. But because I don't speak all this languages, I decided to fix the order of the bot or user notes the words are switched

Reply:
I suppose the change in English will cause all translations to be fuzzied even though they are changed too.